class.jetpack-network.php

```diff
@@ -333,6 +333,7 @@ public function jetpack_sites_list() {
 					}
 
 					wp_safe_redirect( $url );
+					exit;
 					break;
```

Comment:
`break;` isn't necessary here, and may confuse the reader into thinking that execution continues somehow.